hydra-node/test/Hydra/Chain/Direct/StateSpec.hs

```diff
@@ -668,8 +691,8 @@ forAllDecrement' ::
   ([TxOut CtxUTxO] -> UTxO -> Tx -> property) ->
   Property
 forAllDecrement' action = do
-  forAllShrink (genDecrementTx maximumNumberOfParties) shrink $ \(ctx, distributed, st, _, tx) ->
-    let utxo = getKnownUTxO st <> getKnownUTxO ctx
+  forAllShrink (genDecrementTx maximumNumberOfParties) shrink $ \(ctx, distributed, st, utxo', tx) ->
+    let utxo = getKnownUTxO st <> getKnownUTxO ctx <> utxo'
```

Comment:
utxo' is empty and was ignored before.
maybe we can revert this change.

Reply:
We still need to conform to the interface used in tx-cost spec therefore we need to keep this UTxO even if it is not needed for decrements.